src/app/component/ErrorBoundary.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    // You can also log the error to an error reporting service
+    console.error(error, errorInfo);
+    //
+    // TODO hook up to raygun or similar
+    // logErrorToService(error, errorINfo)
+  }
+
+  render() {
+    if (this.state.hasError) {
+      // You can render any custom fallback UI
+      return "";
+    }
+    return this.props.children;
+  }
+}
```

Comment:
Do we want to render nothing if there is an error? My I think the implementation looks fine, just concerned that a blank space in place where the component is supposed to be might not be indicative that an error was thrown. Do we want to keep the error message and display that here? like a "oops something isn't working, please refresh" message?